### PR TITLE
Parse board

### PR DESCRIPTION
**Pull request by kylebisley**
Renamed some but not all of the variables. 
Renamed the file.
Created function int_board to hold the call list for building the pathable_board.
Built and tested it before the pull request.

### CODE REVIEW

File: app/parse_board.py [hunk 1]
```diff
@@ -8,51 +8,68 @@
 LARGER_SNAKE_FUTURE_HEAD = 99
 
 
-def setBoardValues(jData):
+def int_board(converted_data):
     """
-    Converts converted JSON data from the battlesnake engine, to an a_star
-    friendly gameboard.
+    Call list of functions that modify and build the path_board
+    Args: converted_data (dic): Converted json
```
Comment (DimaMzk) [inline app/parse_board.py]:
```suggestion
    Args: 
        converted_data (dict): Converted json
```

File: app/parse_board.py [hunk 2]
```diff
@@ -8,51 +8,68 @@
...
+
+    Returns: A* friendly version of the gameboard
```
Comment (DimaMzk) [inline app/parse_board.py]:
```suggestion
    Returns: 
        A* friendly version of the gameboard
```

File: app/parse_board.py [hunk 3]
```diff
@@ -215,20 +228,23 @@ def display(converted_board, integer_board):
...
-#         pathBoard (array): integer representation of current board 
+#         pathBoard (array): integer representation of current board
 #         xPos(int): the x coordinate of the tile we want to search from
 #         yPos(int): the y coordinate of the tile we want to search from
 #     OBJECT_ORIENTED_TODO: pass in a board object and tile object instead
 #     Returns:
-#         an list of two lists, the first contains all the Tile objects it is possible and viable to path to, the second 
-#         contains all Tile objects that form the walls. Can compare these lists to lists of all food, for example, to get
-#         just food we can path too.
+#         an list of two lists, the first contains all the Tile objects it is
```
Comment (DimaMzk) [inline app/parse_board.py:246]: I'm not sure if this Returns field originated from this commit. But It really needs to be simplified, in terms of readability.

Comment (DimaMzk) [reply app/parse_board.py:246]: ^ Not a change request. But just putting it out there :)

**Review by DimaMzk — CHANGES_REQUESTED** [review]

**Comment by DimaMzk** [comment]
Is this the last bit of pull request for the forseeable future? I'd love to do a pull and just to a bunch of formatting on everything.

**Review by DHervochon — COMMENTED** [review]
Nice job! Makes things a lot clearer to read through.

**Review by DimaMzk — APPROVED** [review]

**Pull request merged**